src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import displayScores from './modules/api.js';
+
+// const submitButton = document.querySelector('.submitButton');
+
+const refreshButton = document.querySelector('#refresh');
+const form = document.querySelector('.inputPlayer');
+const message = document.querySelector('#message');
+const key = 'Fre06jK3qCMjBipi1fVQ';
```

Comment:
[OPTIONAL]

- I will strongly recommend you always make sure that you don't push your secret keys (such as your API key) to GitHub because exposing your secret keys might result in a malicious attack on your project with people using your API key for something else which might impact your project negatively 🤓. 

I will advice you consider storing your API key in an environment variable so that it can be safe 😄

Reply:
I tried using the [documentation](https://github.com/mrsteele/dotenv-webpack/blob/master/README.md). But the key is not accessible while using the environment variable.

Reply:
@asharanjith it's alright . Once you get a better way of using environment variables, please implement it.

I will also check in my end and see articles that might help you. I will reach out when I get something for you. Good job so far 👍🏽

Reply:
@vickymarz Thank you. Appreciate it...